docs/adrs/notifications.md

```diff
@@ -0,0 +1,18 @@
+# Notifications
+
+## Context
+
+superwerker creates an OpsItem in Systems Manager OpsCenter for emails received by the RootMail feature. Without notifications for new OpsItems, users need to manually check the OpsCenter for new items.
+
+## Decision
+
+- Use CloudWatch Events to trigger an AWS Lambda function whenever a new OpsItem is created.
```

Comment:
We decided against using the SNS integration of CreateOpsItem, IIRC. Do we remember why?

Reply:
> OpsCenter / OpsItem do have support for SNS notifications, but the desired SNS topic Arn needs to be provided explicitly whenever an OpsItem is created. Therefore, we decided against this native feature